src/extension.ts

```diff
@@ -490,6 +520,7 @@ export async function activate(context: vscode.ExtensionContext) {
 							stagedJobs.map(({ hash }) => hash as JobHash),
 						);
 
+						// @should rename this to commitJobs?
```

Comment:
maybe not in this PR, but generally let's do such refactorings when the product is more stable